runtime-sdk/src/modules/core/test.rs

```diff
@@ -0,0 +1,36 @@
+use crate::testing::mock;
+
+use super::{
+    API as _,
+    Module as Core,
+};
+
+#[test]
+fn test_use_gas() {
```

Comment:
```suggestion
fn test_use_gas() {
    const MAX_GAS = 1000;
```
this constant is suggested for use below to make the code clearer due to fewer magic numbers